File: src/classes/api/req_res/get-nlog-response.ts
// ˅
'use strict';

import { Nlog } from '@/classes/datas/nlog';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class GetNlogResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    nlog_histories: Array<Nlog>;

    constructor() {
        // ˅
        super()
        this.nlog_histories = new Array<Nlog>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/server/gkill/api/req_res/get_nlog_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type GetNlogResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	NlogHistories []reps.Nlog `json:"nlog_histories"`
}
